packages/react-accordion/config/tests.js

```diff
@@ -0,0 +1,7 @@
+/** Jest test setup file. */
+
+const { configure } = require('enzyme');
+const Adapter = require('enzyme-adapter-react-16');
+
+// Configure enzyme.
+configure({ adapter: new Adapter() });
```

Comment:
nit: If you're not going to use Enzyme in tests, this file can be removed.

Reply:
I'll be adding tests after the Spec review is completed, and the implementation is more robust